oceans/filters.py

```diff
@@ -603,6 +604,16 @@ def pl33tn(x, dt=1.0, T=33.0, mode="valid"):
 
     """
 
+    import cf_xarray  # noqa: F401
+    import xarray as xr
+
+    if isinstance(x, xr.Dataset):
+        raise TypeError("Input a DataArray not a Dataset.")
+
+    if isinstance(x, xr.DataArray):
+        # find dt in units of hours
+        dt = (x.cf["T"][1] - x.cf["T"][0]) * 1e-9 / 3600
```

Comment:
This will get clobbered below, no?

Reply:
Those other lines use an input `dt` though, they just assume 1 hour, then modify the dt according to T.

Reply:
(though I did test with an hourly dataset so...)